macros/SpecName.ejs

```diff
@@ -1151,9 +1139,9 @@ var specList = {
         name : 'WebVTT: The Web Video Text Tracks Format',
         url  : 'https://w3c.github.io/webvtt/'
     },
-    'Web Workers':{ /* This draft has been merged into HTML WHATWG */
-        name : 'Web Workers',
-        url  : 'https://w3c.github.io/workers/'
+    'Web Workers':{
```

Comment:
I've updated the remaining pages. Again, the spam filter probably wasn't happy with the frequent edits when you tried this. That shouldn't happen anymore. 

We could remove the 'Web Workers' entry now.